src/page_body/structure/blocks/page_block_figma_components.pr

```diff
@@ -1 +1,98 @@
-Test 1
\ No newline at end of file
+{[ const block = context /]}
+
+{[ let configuration = exportConfiguration() /]}
+
+{[ let numberOfColumns = block.numberOfColumns /]}
+{[ let variant = block.variantKey /]}
+
+{[ if (variant === "grid") ]}
+    <div class="content-block content-block--figma-components {{ getVariantClass(variant) }} layout-columns-{{ numberOfColumns }}" id="{{ getSearchIDString(block.id) }}">
+        {[ for selectedComponent in block.selectedFigmaComponents ]}
+            {[ const component = ds.designComponentById(selectedComponent.figmaComponentId) /]}
```

Comment:
It is probably possible that component won't be present anymore, we should handle that case